tstest.ts

```diff
@@ -70,6 +70,11 @@ import Pulsar = require('./index');
     token: 'foobar',
   });
 
+  const authBasic: Pulsar.AuthenticationBasic = new Pulsar.AuthenticationBasic({
+    username: 'basic.username',
+    password: 'basic.password',
+  });  
```

Comment:
There is unnecessary whitespace at the end of the line.
```suggestion
  });
```